wfe/wfe.go

```diff
@@ -373,6 +376,9 @@ func (wfe *WebFrontEndImpl) Directory(ctx context.Context, logEvent *requestEven
 		// field on a User-Agent header that doesn't start with 'LetsEncryptPythonClient'
 		directoryEndpoints["key-change"] = rolloverPath
 	}
+	if features.Enabled(features.GreaseDirectory) {
+		directoryEndpoints[core.RandomString(8)] = "grease"
```

Comment:
One idea I just had: It might be a good call to put an issue URL or some other information in the value that would help shorten the gap between "Why is my client breaking" and "Oh, it's directory change intolerant"